ocs_samples/library_samples/Python3/ocs_sample_library_preview/Dataviews.py

```diff
@@ -342,8 +355,12 @@ def getDataInterpolated(
         )
 
         next_page = response.headers.get("NextPage", None)
-        continuation_token = next_page[next_page.find("&continuationToken=")+19:] if next_page is not None else None
-        
+        continuation_token = (
+            next_page[next_page.find("&continuationToken=") + 19 :]
```

Comment:
19?  is this interval -1

Reply:
This is to extract the continuationToken which is always 19 characters further than the index where this substring starts, plus is always the last piece of information returned in the next page URL.

Reply:
More robust code added